dirty.html

```diff
@@ -1,49 +1,36 @@
 <html>  
   <head>
-    <h1>WDI London September 2013
+    <h1>WDI London September 2013</h1>
     <meta value="description" content="Web development immersive ">
   </head>
   <div class="body">
-    <div>
-        <div>
-        </div>
-        <div>
-          <div>Web Development Immersive <div>General&nbsp;Assembly:&nbsp;Autumn/Winter&nbsp;2013</div></div>
+          <h1>Web Development Immersive </h1>
+          <h2>General&nbsp;Assembly:&nbsp;Autumn/Winter&nbsp;2013</h2>
```

Comment:
no need for encoding spaces as `&nbsp`, you can just put:

`<h2>General Assembly Autumn/Winter 2013</h2>`